core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -932,7 +936,24 @@ public static String getDefaultOncoprintView() {
         }
         return defaultOncoprintView.trim();
     }
-
+    
+    public static boolean useCustomDriverAnnotations() {
+        String useCustomDriverAnnotations = properties.getProperty(USE_CUSTOM_DRIVER_ANNOTATIONS, "true");
+        if (getBinaryCustomDriverAnnotationMenuLabel()==null && getClassCustomDriverAnnotationMenuLabel()==null)
```

Comment:
I think you will need to compare to empty string as well, since the variable could be there, but just set to nothing. E.g. 
```
oncoprint.custom_driver_annotation.class.menu_label=
```